editoast/src/views/infra/mod.rs

```diff
@@ -158,6 +173,60 @@ async fn create(db_pool: Data<DbPool>, data: Json<InfraName>) -> Result<impl Res
     Ok(HttpResponse::Created().json(infra))
 }
 
+/// Duplicate an infra
+#[post("/clone")]
```

Comment:
```suggestion
#[post("/{infra}/clone")]
```
~otherwise `/infra/1/clone/` and `/infra/clone/1` will work but not be documented in openapi.yml~ never-mind it works in all cases with both syntax

Reply:
So, should I update the openapi ?

Reply:
nope sorry i delete my comment

Reply:
Thanks :)